relayer/src/channel.rs

```diff
@@ -756,14 +756,12 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Channel<ChainA, ChainB> {
             _ => State::Uninitialized,
         };
 
-        let version = version::resolve(ResolveContext::Other, self)?;
-
         let dst_expected_channel = ChannelEnd::new(
             highest_state,
             self.ordering,
             counterparty,
             vec![self.dst_connection_id().clone()],
-            version,
+            Version::from(""),
```

Comment:
Can we extract this into a `Version::empty()` method?

Reply:
908c3e86